pkg/chunk/storage/factory.go

```diff
@@ -42,7 +64,7 @@ func NewStorageClient(cfg Config, schemaCfg chunk.SchemaConfig) (chunk.StorageCl
 		}
 		return aws.NewStorageClient(cfg.AWSStorageConfig, schemaCfg)
 	case "gcp":
-		return gcp.NewStorageClient(context.Background(), cfg.GCPStorageConfig, schemaCfg)
+		return gcp.NewStorageClient(context.Background(), cfg.GCPStorageConfig, schemaCfg, bigtableColumnKey)
```

Comment:
Move the bigtableColumnKey flag into GCPStorageConfig so we don't have to pass it everywhere?

Reply:
Done. But this was what I did initially, you were not okay re-writing the config then. Not sure of a method where we could do this without either making it an argument or re-writing config.

Reply:
I could make `*.NewStorageClient` return `[]chunk.StorageOpt` but don't think that is clean.